server/services/participants.js

```diff
@@ -141,9 +140,7 @@ const confirmParticipantInterest = async (id) => {
     })
     .find({
       'body.interested IS DISTINCT FROM': 'withdrawn', // "IS DISTINCT FROM" = "!=" but includes null
-      'body.userUpdatedAt::TIMESTAMP <': sixWeeksAgo,
```

Comment:
I would think we want to keep this check, why is it removed?

Reply:
The updated at column can be changed without the users action. It is a terrible way of measuring interest. 
It leads to edge cases in testing as well.  Having a valid OTP should be evidence enough.